deno-app/main.ts

```diff
@@ -0,0 +1,52 @@
+import { serve } from "https://deno.land/std@0.173.0/http/server.ts";
+import { Deta } from "https://esm.sh/deta@1.1.0";
+
+const deta = Deta(Deno.env.get("DETA_PROJECT_KEY")!);
```

Comment:
manually grabbing the key should be not encouraged. just explain it
```suggestion
// Deta will automatically grab the key `DETA_PROJECT_KEY` from the environment.
const deta = Deta();
```
why not also just import `Base` and `Drive` from the lib directly as we did with python?

Reply:
In my testing it wouldn't automatically get the key, so I had to fetch it manually.

Reply:
Issue somehow fixed itself since last night ¯\_(ツ)_/¯